src/TemplateLibrary/Buttons/index.js

```diff
@@ -0,0 +1,58 @@
+/* React */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* Styling */
+import styled from 'styled-components';
+import { Button } from 'semantic-ui-react';
+
+const UploadButton = styled.a`
```

Comment:
I think it makes more semantic sense for this to be the html element `button` instead of `a`. this would also negate the need to use `href="javascript:void(0);"`. It may mean you'll need to edit the styling again though.